app.py

```diff
@@ -81,6 +84,70 @@ def get_user_by_id(user_id: str):
     return json.dumps(user)
 
 
+"""
+adds new known image
+"""
+@app.route('/known', methods=['POST'])
+def add_known():
+    data = request.get_json("data")
+    img = data['img']
+    name = data['name']
+    known = {
+        "name": name,
+        "img": img
+    }
+    result = db.known.insert_one(known)
+    return make_response()
+
+
+"""
+returns all known images
+"""
+@app.route('/known', methods=['GET'])
+def get_all_known():
+    entries = []
+    cursor = db.known.find({})
+    for document in cursor:
+        document['_id'] = str(document['_id'])
+        entries.append(document)
+
+    response = jsonify(entries)
+    response.headers.add('Access-Control-Allow-Origin', '*')
+
+    return response
+
+
+"""
+adds new unknown image
+"""
+@app.route('/unknown', methods=['POST'])
+def add_unknown():
+    data = request.get_json("data")
+    img = data['img']
+    known = {
+        "img": img
```

Comment:
I might add in the time stamp so that way we can kind of track unknown people. might be beneficial to have later on